### PR TITLE
add into and snap

### PR DESCRIPTION
**Pull request by ericfode**
Docs for https://github.com/urbit/urbit/pull/1974

### CODE REVIEW

File: reference/library/2b.md [hunk 1]
```diff
@@ -931,6 +931,77 @@ Produces a list of `b` appended to `a`.
     ~[1 2 3 4]
 ```
 
+---
+### `++snap`
+
+Replace Item at index
```
Comment (syzygyzer) [inline reference/library/2b.md]: Item shouldn't be capitalized

File: reference/library/2b.md [hunk 2]
```diff
@@ -931,6 +931,77 @@ Produces a list of `b` appended to `a`.
...
+
+Accepts a `++list` `a`, an atom `b`, and a noun `c`, producing the list of `a` with the item at index `b` replaced with `c`.
```
Comment (syzygyzer) [inline reference/library/2b.md]: `++list` should be `list`

File: reference/library/2b.md [hunk 3]
```diff
@@ -931,6 +931,77 @@ Produces a list of `b` appended to `a`.
...
+
+```
+    > (snap (limo ~[2 3 4]) 1 11)
+    ~[2 11 4]
+```
+
+---
+### `++into`
+
+Insert Item at index
```
Comment (syzygyzer) [inline reference/library/2b.md]: Item -> item

File: reference/library/2b.md [hunk 4]
```diff
@@ -931,6 +931,77 @@ Produces a list of `b` appended to `a`.
...
+
+Accepts a `++list` `a`, an atom `b`, and a noun `c`, producing the list of `a` with the item `c` inserted at index `b`.
```
Comment (syzygyzer) [inline reference/library/2b.md]: `++list` -> `list`

**Comment by ericfode** [comment]
Thank you!

**Comment by joemfb** [comment]
@rmariani, probably better to wait for the additions to be merged next time.

In fairness, this isn't anything like a common issue. @ericfode has gone above and beyond documenting new gates while adding them. We haven't usually done that, but should strive to ....

**Comment by rmariani** [comment]
Whoops, thought everything was added already. My mistake!

**Comment by joemfb** [comment]
No need to revert or anything, those changes will go in -- the only issue is some details in the tests.

**Pull request merged**